test/karma.conf.js

```diff
@@ -92,6 +100,11 @@ module.exports = (config) => {
   if (process.env['TRAVIS']) {
     let buildId = `TRAVIS #${process.env.TRAVIS_BUILD_NUMBER} (${process.env.TRAVIS_BUILD_ID})`;
 
+    if (process.env['TRAVIS_PULL_REQUEST'] === 'false') {
+      config.preprocessors['dist/@angular/material/**/!(*spec).js'] = ['coverage'];
```

Comment:
I intentionally didn't do it because there can be also `-spec.js` files. Improved the glob by now checking for `-|.`